jdi-light/src/test/java/io/github/epam/listtests/DataListTests.java

```diff
@@ -0,0 +1,24 @@
+package io.github.epam.listtests;
+
+import com.epam.jdi.light.elements.complex.DataList;
+import io.github.epam.ListTestsInit;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class DataListTests extends ListTestsInit {
+    @Test
+    public void jdiDataListTest() {
+        dataList = new DataList();
+        dataList.list().setWebElements(list);
+        assertEquals(dataList.get(1).getText(), element0Text);
+        assertEquals(dataList.get(2).getText(), element1Text);
+    }
+
+    @Test (expectedExceptions = RuntimeException.class)
+    public void jdiDataListNullTest() {
+        dataList = new DataList();
+        dataList.list().setWebElements(list);
+        dataList.get(0);
```

Comment:
there is no any check here but it's a test

Reply:
Here is @Test (expectedExceptions = RuntimeException.class) above this method;)

Reply:
> there is no any check here but it's a test

Here is @test (expectedExceptions = RuntimeException.class) above this method;)